eng/docgeneration/docindex.yml

```diff
@@ -0,0 +1,119 @@
+                  Write-Host "Write $($ProjectName.Name)"
+                  Add-Content -Path "$($YmlPath)/$($Dir.Name).md" -Value "#### $($ProjectName.BaseName)"
+              }
+          }
+          foreach ($Dir in $ServiceList)
+          {
+              Write-Host "Write Mgmt Header"
+              $PkgList = Get-ChildItem $Dir.FullName -Directory | Where-Object -FilterScript {$_.Name -match ".Management."}
+              if (($PkgList | Measure-Object).count -eq 0)
+              {
+                  continue
+              }
+              Add-Content -Path "$($YmlPath)/$($Dir.Name).md" -Value "# MANAGEMENT"
+              Add-Content -Path "$($YmlPath)/$($Dir.Name).md" -Value "---"
+              foreach ($Pkg in $PkgList)
+              {
+                  $ProjectName = Get-ChildItem "$($pkg.FullName)\src\*" -Include *.csproj
+                  Write-Host "Write $($ProjectName.Name)"
+                  Add-Content -Path "$($YmlPath)/$($Dir.Name).md" -Value "#### $($ProjectName.BaseName)"
+              }
+          }
+          New-Item -Path "$(Build.SourcesDirectory)/docfx_project" -Name "toc.yml" -Force
+          Add-Content -Path "$(Build.SourcesDirectory)/docfx_project/toc.yml" -Value "- name: Azure SDK for NET APIs`r`n  href: api/`r`n  homepage: api/index.md"
+          Copy-Item "$(Build.SourcesDirectory)/README.md" -Destination "$(Build.SourcesDirectory)/docfx_project/api/index.md" -Force
+          Copy-Item "$(Build.SourcesDirectory)/*.md" -Destination "$(Build.SourcesDirectory)/docfx_project/api" -Force
+        displayName: Create main index and navigation toc.yml, copy over readme.
+      - pwsh: ls
+        workingDirectory: $(Build.SourcesDirectory)/docfx_project/api
+      - task: CmdLine@2
+        displayName: Build Doc Content
+        inputs:
+          script: $(Build.BinariesDirectory)/docfx/docfx.exe build
+          workingDirectory: $(Build.SourcesDirectory)/docfx_project/
+          failOnStderr: true
+      - pwsh: |
+          Copy-Item "$(Build.SourcesDirectory)/eng/docgeneration/assets/logo.svg" -Destination "$(Build.SourcesDirectory)/docfx_project/_site/" -Force
+          Copy-Item "$(Build.SourcesDirectory)/eng/docgeneration/assets/toc.yml" -Destination "$(Build.SourcesDirectory)/docfx_project/_site/" -Force
+        displayName: Replace site assets
+      - task: UsePythonVersion@0
+        displayName: 'Use Python 3.6'
+        inputs:
+          versionSpec: '3.6'
+      - template: eng/pipelines/templates/scripts/mashup-doc-index.yml@azure-sdk-tools
+        parameters:
+          SourceDirectory: $(Build.SourcesDirectory)
+      - task: CopyFiles@2
+        displayName: Copy HTML to Artifacts Directory
+        inputs:
+          sourceFolder: $(Build.SourcesDirectory)/docfx_project/
+          content: '**\*'
+          targetFolder: $(Build.ArtifactStagingDirectory)/docfx_project
+          overWrite: true
+      - task: PublishPipelineArtifact@0
```

Comment:
Is there a goal to add the publishing of these as part of a unified pipeline as opposed to using a separate Release pipeline?

Reply:
I'll add it as a step in the main doc generation.

Reply:
Actually I have to check with @scbedd to see how this fits into the unified pipeline doc publishing.